### DISCUSSION TITLE
Unathi/Cindarite/Lizard/etc. Lore Proposal

### DISCUSSION THREAD

**Discussion by SigDoesCode** [Equinox-SS13/equinox-sojourn#94] [Ideas]
General Gist:
Apex predator species domesticated really early on in evolution by two factions constantly warring on a singular planet. After the war ended they're left on the polluted, toxic, diseased, irradiated, stormy, etc. planet and evolve over a few million years before harnessing tech left behind and combining it with their own developments. They're thick, strong, slow tanks, big lizards used to warfare. Think a combination of the Fremen from Dune and Imperial Space Marines in aesthetic, rubes and armor plates on big stompy people, meant to withstand storms and harsh environments. 
 
Generalized resistance to disease and toxins, due to evolving on a hellworld. Resistance to heat because of their planet's atmosphere. Cold fucks them up though, they're not used to it. A bit of innate armor to represent their bulky nature. Slowdown innately because huge lizard. No slowdown from clothes and armor from the domesticated pack animal bit: they wear heavy armor instinctively (he will know your ways as if born into them...)

General look is thinking BIG lizards, like komodo dragons, alligators, monitor lizards, etc. except like, anthropomorphic and 7-8' tall.

General History: 
Unathi/cindarite/etc. (name goes here, not come up with one yet if we want to change it) developed on a planet previously used by two warring factions as a "battle world," named Moghes (name TBD, going with a placeholder for now). These two civilizations, now lost to time, warred on the [species] home planet instead of each others planets to prevent collateral damage to each others' territories due to the brutality of the war itself. 

Moghes itself is a ruined world, marked with craters from orbital munitions drops, artillery, explosives, and similar. Temperature on Moghes is higher than Earth, though the atmosphere is drier, but not dry enough to be considered a desert planet. Rust, dust, and sand storms mark the surface, especially on older battlefields. The atmosphere is comparable to Earth, but the pollutants released by constant warfare (chemical, biological, conventional, and, previously, nuclear) resulted in a mixture hazardous to most life for long periods.  Due to the constant war, most fauna was wiped out, leaving a species of large, heavy reptiles to prosper once the war was over. These reptiles were once used as pack animals by both sides, according to information found in the various bunkers dotting the planet. Similar bunkers contained weaponry, schematics, and technology that allowed the [species] to develop rather quickly once they developed the ability to decipher some of the information. Due to the damage the archives sustained during combat there was not enough information to fully catapult [species] into the space age. Instead, they ended up with a technology level akin to most galactic species, mixing this "progenitor tech" and their own developments. 

Representing this in Code: 
Give them heat resistance
Keep their current abilities, as they're rather weak if thematic
Give them a small amount of innate armor
Keep their slowdown
Give them cold weakness/vulnerability

Currently, Unathi have a set of three perks: Uncanny Resilience, which gives them 5u spaceacillin every 30 minutes, making them basically immune to disease; Purge Toxins, a perk that allows them to inject 5u of Cindpetamol, giving them extreme toxin reduction at the cost of slowdown and organ damage; and Second Skin, a perk that prevents slowdown from clothing or armor. This sets them up to be a sort of "inevitable object" because they can be heavily armored and immune to toxin and disease, only going down to direct damage. 

In practice, this isn't super great. Maybe giving them a bit more armor might make up for this, because currently the big part of the game is not getting hit in the first place. Regardless, it sets them up as a sort of indestructible object.

**Comment by SigDoesCode** [comment]
Comments:
Salvager culture/aesthetic
Possibly ambush predators
Possibly pushed around the planet by a combination of environmental effects and automated war machines that are still around/periodically reactivated by salvagers
The various "jobs" (medic, salvager, etc.) become a species background pick

**Comment by Vlada3333** [comment]
Interesting, thanks!

**Comment by SigDoesCode** [comment]
old skrell empire as one of the members of the war?
